name-registry/tests/utils/mod.rs

```diff
@@ -0,0 +1,40 @@
+pub mod abi;
+
+use fuels::{prelude::*, tx::ContractId};
+
+// Load abi from json
+abigen!(NameRegistry, "out/debug/name-registry-abi.json");
+
+pub async fn get_contract_instance() -> (NameRegistry, ContractId, WalletUnlocked, WalletUnlocked) {
```

Comment:
```suggestion
pub async fn setup() -> (NameRegistry, ContractId, WalletUnlocked, WalletUnlocked) {
```

This is doing a lot more than just returning a contract instance and I suggest you add more to this, as per comment(s) above